shared/src/commonMain/kotlin/com/example/travelapp_kmp/listing/MainScreen.kt

```diff
@@ -268,7 +268,10 @@ internal fun ImageSlider(
                 modifier = Modifier
                     .width(width = (width * 0.62).dp)
                     .aspectRatio(ratio = (295.0 / 432.0).toFloat())
-                    .clip(RoundedCornerShape(20.dp)),
+                    .clip(RoundedCornerShape(20.dp))
+                    .clickable { onDetailsClicked(
+                        touristPlace
+                    ) },
```

Comment:
This will make the whole card clickable. We need to navigate only when discover place is clicked.

Reply:
Okay brother, I thought it would be great if you can click the whole card instead of trying to click on the small discover place. I am reverting it and will update. Thanks for the feedback brother. Much appreciated.